redLayerModule.py

```diff
@@ -299,7 +301,13 @@ def initGui(self):
             callback=self.loadAction,
             parent=self.iface.mainWindow(),
             object_name='mLoadSketchesFromFile')
-        self.canvasButton.setMenu(self.canvasMenu())
+        # create explicit QToolButton for canvas action (Qt5 & Qt6)
+        canvas_toolbutton = QToolButton(self.iface.mainWindow())
+        canvas_toolbutton.setDefaultAction(self.canvasButton)
+        canvas_toolbutton.setMenu(self.canvasMenu())
+        canvas_toolbutton.setPopupMode(QToolButton.InstantPopup)
```

Comment:
```suggestion
        canvas_toolbutton.setPopupMode(QToolButton.ToolButtonPopupMode.InstantPopup)
```

I'm an idiot, sorry. Obviously that enum needs the full qualifier in Qt6! :D